tests/python/frontend/onnx/test_forward.py

```diff
@@ -1116,6 +1117,15 @@ def test_inception():
 # def test_shufflenetv2():
 #     check_torch_conversion(torchvision.models.shufflenetv2, (1,3,224,224))
 
+def test_sign():
+    def Sign_x(x):
+        return np.sign(x)
+    _test_onnx_op_elementwise((2, 4, 5, 6),
```

Comment:
should we have a mix of positive and negative values?

Reply:
(2, 4, 5, 6) is the shape. The test will create a tensor with elements between -1 and 1, so it will have negative values.